pages/implementation/reference/java.mdx

```diff
@@ -0,0 +1,222 @@
+
+You can also <Link passHref href="/explore-tracking-plan/download-or-copy-avo-file-manually"><a>download the file manually</a></Link> from your Avo workspace.
+
+#### Step 2. Initialize Avo
+
+Initialize Avo by calling the `initAvo` method before tracking
+
+```java
+Avo.initAvo(Avo.AvoEnv.DEV, /*, other parameters depending on your tracking plan setup*/);
+```
+
+The actual parameters depend on your tracking plan setup, see the parameters explanation in <Link passHref href="/implementation/reference/java#initAvo"><a>the reference below</a></Link>.
+
+#### Step 3. Call Avo functions to track your product usage
+
+Every event in your tracking plan, marked with the "Implement with Avo" checkbox, gets a function in the generated code, named according to the event name, in camelCase.
+
+[//]: # 'TODO Future: Add link here to the "Implement with Avo" doc in "Tracking Plan Management"'
+
+For example, if you have a "Signup Start" event defined like this in Avo:
+
+<center style={{ marginBottom: 16 + 'px' }}>
+  <img
+    style={{ height: 600 + 'px' }}
+    src={require('../../../images/signup-start-ui.png')}
+    alt="Event 'Signup Start' defined in Avo with referral string property and implement with Avo check"
+  />
+</center>
+
+You'll be able to call it like this from the generated code
+
+```java
+Avo.signupStart("direct")
+```
+
+> Notice, that you are not passing the System property with the call. System properties are defined on the init step and then automatically included with all events.
+> You can update the system properties with `setSystemProperties` function.
+
+#### Step 4. Verify the implementation
+
+Use the <Link passHref href="/workspace/tracking-plan/implementation-status"><a>Implementation status</a></Link> in your Avo workspace and the <Link passHref href="/implementation/inspector/sdk/android"><a>Avo Inspector</a></Link> to verify that your implementation is correct. If you don't want to use the Avo Inspector you can use the <a href="https://github.com/avohq/android-analytics-debugger">standalone visual debugger</a>.
+
+### Reference
+
+#### initAvo
+
+```java
+public static void initAvo(AvoEnv env,
+    // Other parameters may not be present, depending on your tracking plan
+    Integer systemProprty0, Boolean systemProperty1,
+    ICustomDestination customDestDestination,
+    boolean strict = true, boolean noop = false,
+    application: Application, context: Context
+    );
+```
+
+There is also an init method that accepts an instance of the Avo Inpsector, use it if you have Avo Inspector set up in your project, so Avo Functions will automatically report invocations to the Avo Inspector.
+
+```java
+public static void initAvoWithInspector(AvoEnv env,
+    Object avoInspector,
+    // Other parameters may not be present, depending on your tracking plan
+    // Same parameters, as in `initAvo`
+    Integer systemProprty0, Boolean systemProperty1,
```

Comment:
```suggestion
    Integer systemProperty0, Boolean systemProperty1,
```

Reply:
This typo is all over the reference, good catch 😄